src/build_gsi.sh

```diff
@@ -0,0 +1,27 @@
+#! /usr/bin/env bash
+set -eux
+
+source ./machine-setup.sh > /dev/null 2>&1
+cwd=`pwd`
+
+USE_PREINST_LIBS=${USE_PREINST_LIBS:-"true"}
+if [ $USE_PREINST_LIBS = true ]; then
+  export MOD_PATH=/scratch3/NCEPDEV/nwprod/lib/modulefiles
+else
+  export MOD_PATH=${cwd}/lib/modulefiles
+fi
+
+# Check final exec folder exists
+if [ ! -d "../exec" ]; then
+  mkdir -p ../exec
+fi
+
+cd gsi
+
+if [ "$target" = "jet" ] ; then
+  ./ush/build.comgsi
+elif [ "$target" = "hera" ] ; then
+  ./ush/build.comgsi
```

Comment:
Can this be an "or" expression so that we do not duplicate code? The same for build_rrfs_utl.sh.

Reply:
Done